libr/bin/p/bin_symbols.c

```diff
@@ -193,7 +193,7 @@ static RBinSymbol *bin_symbol_from_symbol(RCoreSymCacheElement *element, RCoreSy
 	return sym;
 }
 
-static RCoreSymCacheElement *parseDragons(RBinFile *bf, RBuffer *buf, int off, int bits) {
+static RCoreSymCacheElement *parseDragons(RBinFile *bf, RBuffer *buf, int off, int bits, char * file_name) {
```

Comment:
const char*

Reply:
why `const`? the `file_name` buffer is going to be owned (and later free'd) from the element we're going to create here if everything goes well. If i `const` it here, i'd have to cast the `const` away later in the code, or am i missing something?

Reply:
Oh i see, then its better to use the R_OWNED empty macro just as sugar to make that clear

Reply:
how is the `R_OWN` intended to be used here? at the level of the single argument? or the whole function?

like this:
```c
static RCoreSymCacheElement *parseDragons(RBinFile *bf, RBuffer *buf, int off, int bits, R_OWN char *file_name) {
```

or like this:
```c
R_OWN  static RCoreSymCacheElement *parseDragons(RBinFile *bf, RBuffer *buf, int off, int bits, char *file_name) {
```

?